### ISSUE TITLE
fundamental tutorials

### ISSUE THREAD

**Issue by wangjiawen2013** [huggingface/candle#1774]
Hi,
I haven't found fundamental tutorials on Candle, it looks like the target audience of Candle are veterans on deep learning . Could anyone provide some tutorials on candle (or even basic knowledge on machine learning) for beginners to follow step by step ?

**Comment by danielclough** [comment]
I am also an AI/ML noob, so I would also love this.

I have been building example projects that push my limits in order to engage with the code and learn from doing.

There is a lot that can be done with candle without any knowledge of deep learning at all!

I suggest playing with the example code until someone makes some tutorials for us to learn from. :hugs:

**Comment by KGrewal1** [comment]
https://github.com/ToluClassics/candle-tutorial is this sufficient or do you mean from a baseline of no prior ML / linalg whatsoever?

**Comment by wangjiawen2013** [comment]
This one is good.  Though I have some understanding on classic machine learning (perceptron, SVM, bayes classifier and so on), it's also welcome if you give some basic knowledge on **deep learning**, such as neuron network/tensor

**Comment by danielclough** [comment]
I could use an ELI5. :hugs:
